templates/csharp/api-plugin-from-scratch-oauth/infra/azure.bicep.tpl

```diff
@@ -81,6 +80,7 @@ var aadApplicationIdUriWithDomain = 'api://${functionApp.properties.defaultHostN
 {{/MicrosoftEntra}}
 
 // Configure Azure Functions to use Azure AD for authentication.
+var clientIdForTGS = 'ab3be6b7-f5df-413d-ac2d-abf1e3fd9c0b'
```

Comment:
Do we wrap this line with {{#MicrosoftEntra}}?

Reply:
Yes, let me update it, thanks!